internal/acceptance/data_volume_test.go

```diff
@@ -0,0 +1,53 @@
+package acceptance
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func checkDataSourceVolume(t *testing.T) func(s *terraform.State) error {
+	return func(s *terraform.State) error {
+		r, ok := s.Modules[0].Resources["data.databricks_volume.this"]
+		require.True(t, ok, "data.databricks_volume.this has to be there")
+		assert.Equal(t, s.Modules[0].Resources["databricks_volume.this"].Primary.ID, r.Primary.Attributes["full_name"])
```

Comment:
Check is failing with:

```
        	            	expected: "sandboxtjecbdagljicg.thingstjecbdagljicg.volume_data_source_test"
        	            	actual  : ""
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-sandboxtjecbdagljicg.thingstjecbdagljicg.volume_data_source_test
```

Most probably because of this line:

```
r.Primary.Attributes["full_name"]
```

because `full_name` is inside the nested block.

Reply:
assertion was removed